libraries/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj

```diff
@@ -34,4 +34,8 @@
 		<ProjectReference Include="..\Microsoft.Bot.Connector\Microsoft.Bot.Connector.csproj" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <Folder Include="Inspection\" />
```

Comment:
Is this needed? The "Adapters" folder doesn't have such an include - not clear if this needs it or not.

Reply:
not needed but nice:-) this is self contained feature, I'd also added sub folders for integration. just keeps the code organised in VS not visible in the product